src/rt/lifetime.d

```diff
@@ -1120,7 +1120,7 @@ extern (C) void[] _d_newarraymiTX(const TypeInfo ti, size_t[] dims)
  * Allocate an uninitialized non-array item.
  * This is an optimization to avoid things needed for arrays like the __arrayPad(size).
  */
-extern (C) void* _d_newitemU(in TypeInfo _ti)
+extern (C) void* _d_newitemU(in TypeInfo _ti) pure nothrow
```

Comment:
The corresponding `_d_newarray*` functions are `pure nothrow` already: https://github.com/dlang/druntime/blob/b7391cd28b5d1fff9ffa9d6e675f08c4b4b5385f/src/rt/lifetime.d#L936-L1044